hw/ip/dcd/doc/checklist.md

```diff
@@ -115,8 +115,8 @@ Review        | Signoff date            | Not Started |
 
  Type         | Item                                  | Resolution  | Note/Collaterals
 --------------|---------------------------------------|-------------|------------------
-Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Not Started | [DCD DV Plan]({{<relref "hw/ip/dcd/doc/dv_plan" >}})
-Documentation | [DV_PLAN_COMPLETED][]                 | Not Started | [DCD Testplan]({{<relref "hw/ip/dcd/doc/dv_plan/index.md#testplan" >}})
+Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Not Started | [DCD DV Plan]()
+Documentation | [DV_PLAN_COMPLETED][]                 | Not Started | [DCD Testplan]()
```

Comment:
This would be:
```
Documentation | [DV_PLAN_COMPLETED][]                 | Not Started | [DCD DV Plan]({{<relref "hw/ip/dcd/doc/dv/index.md#dv-plan" >}})
```